packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -9,16 +9,17 @@
                 <path d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z"/>
             </svg>
 
-            <div v-if="link.path">
-                <router-link :to="link.path" @click="setData" :class="{'router-link-active': link.imActive, 'router-link-exact-active': link.imActive}">{{link.title}}</router-link>
+             <div v-if="link.path">
+                <router-link :to="link.path" exact @click="setData" class="tree-nav-link">{{link.title}}</router-link>
```

Comment:
What does `exact` do?

Reply:
Here is link for Vue documentation  - https://router.vuejs.org/api/#exact

In general, `exact` will apply `router-link-active` class (default class name provided by Vue) ONLY if the whole current URL will be exactly equal to `:to` directive of `<router-link>`

E.g. we have such URL structure: 

- /parent/
- /parent/first-child/
- /parent/second-child/
- .....
- /parent/nth-child/

`<router-link>` provides `router-link-active` class and URL tracking feature from the box by default. If we will not use `exact` directive, `<router-link>` will treat /parent link as active link, even if current active link is /parent/nth-child, because in provided structure all routes from /first-child/ to /nth-child/ will be treated as siblings and /parent/ will be treated as parent.  

This why there was `router-link-active` class added to each first item in each sidebar sections (Concepts, Languages and SDKs, References). 

Also, due to out-of-the box default `<router-link>` features there is no need to manually add or remove classes to/from this component in Vue, see issue with dual class sets in description of PR.